lessons/templates/lesson_request_delete.html

```diff
@@ -1,6 +1,14 @@
+{% extends 'partials/navbar.html' %}
+{% block content %}
+
```

Comment:
This template you could add more meaning to it. For example, you could make it a 'ARE YOU SURE YOU WANT TO DELETE THIS REQUEST?' instead of just having a dull button